docs/app/Examples/elements/List/Content/ListIconExample.js

```diff
@@ -0,0 +1,27 @@
+import React, {Component} from 'react';
+import {List, ListItem} from 'stardust';
+
+export default class ListIconExample extends Component {
+  render() {
+    const helpIcon = <i className='help icon' />;
+    const triangleIcon = <i className='right triangle icon' />;
+
+    return (
+      <List>
+        <ListItem
+          image={helpIcon}
+          header='Floated Icon'
+          description='This text will always have a left margin so it sits alongside the icon'
+        />
+        <ListItem
+          image={triangleIcon}
+          header='Icon Alignment'
+          description='Floated icons are by default top aligned'
+        />
+        <ListItem image={helpIcon}>
+          Inline Text
```

Comment:
Nice job including this example!  I was hoping to see it.

I think we should use React `children` terminology to describe this usage.  Also, I'd like to place it right at the top.  How about duplicating the very first example set, but with a description that says something like, "<code>ListItems</code> also render <code>children</code> as their description."?

Here is the example I'm referring to copying:

![image](https://cloud.githubusercontent.com/assets/5067638/10823305/515c24d8-7e18-11e5-8848-0cddb358708a.png)